cmd/argocd/commands/util.go

```diff
@@ -0,0 +1,48 @@
+package commands
+
+import (
```

Comment:
Nitpicking. Please run `goimports -w cmd/argocd/commands/util.go`. We usually automatically run goimports on file save, so next person might get unexpected changes.